lua/spectre_common.lua

```diff
@@ -404,6 +404,7 @@ local function fetch_from_cache(cassandra_helper, id, uri, destination, cache_na
     local cache_status = cached_value['body'] ~= nil and 'hit' or 'miss'
     local dims = {{'namespace', destination}, {'cache_name', cache_name}, {'cache_status', cache_status}}
     metrics_helper.emit_timing('spectre.fetch_body_and_headers', (socket.gettime() - start_time) * 1000, dims)
+    metrics_helper.emit_counter('spectre.hit_rate', dims)
```

Comment:
I'm not a big fan of the name "hit_rate" for a counter which emits only hits or misses. "cache_fetch" or "cache_fetch_count" would probably be a better name?

Reply:
Good point. I chose this name since we already emit `bulk_hit_rate`, which is the same thing but for bulk endpoints.

If you feel strongly about this I can change both

Reply:
Nah don't worry about it. Internally we'll end up looking at dashboards. This name only matters to the maintainers of said dashboards, so it's probably not worth changing at this point.